src/gseproject/EXEplorer/models.py

```diff
@@ -1,7 +1,41 @@
+from django.conf import settings
 from django.db import models
-from django.contrib.auth.models import User
 # Create your models here.
 
-class UserProfile(models.Model):
-    user = models.OneToOneField(User, on_delete=models.CASCADE)
-    coins = models.IntegerField(default=0)
\ No newline at end of file
+class Player(models.Model):
+    user = models.CharField(settings.AUTH_USER_MODEL, on_delete = models.CASCADE, verbose_name = "User", related_name = "players")
```

Comment:
Is this ther username field, as it's only a string? If so can it be renammed to username

Reply:
edited